packages/ui-kit/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@testring/ui-kit",
+  "version": "0.3.6",
+  "main": "./dist/index.bundle.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ringcentral/testring.git"
+  },
+  "author": "RingCentral",
+  "license": "MIT",
+  "scripts": {
+    "storybook": "start-storybook -p 9001 -c storybook"
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "^3.4.10",
```

Comment:
consider use fixed version for packages. i.e. `"3.4.10"` vs `"^3.4.10"`. flexible version may cause a bugs.